### ISSUE TITLE
allow to set multiple different key name for each attachement

### ISSUE THREAD

**Issue by ahmadnassri** [ahmadnassri/app-restconsole#38]
request from the chrome store.

**Comment by evertharmeling** [comment]
Somewhat related to this issue, it would be neat if you could add a 'clear up'  / 'delete' button for the the attachment(s), so you don't have to refresh the REST Console to be able to do another request which does not involve the attachment(s).

Also 2 minor related typo issues:
- "Attachements" should be "Attachments"
- "Choose one or more files to upload. By default, file names will be used as paremter keys." should be "parameter keys"

**Comment by ahmadnassri** [comment]
@evertharmeling agreed, should be able to remove the attachments just like the body parameters

and I'll fix the typos as well :)
